logger/logger.go

```diff
@@ -85,11 +86,29 @@ func New(printer Printer) *Logger {
 		p:      printer,
 		prefix: DefaultPrefix,
 	}
-	l.SetLevel(ErrorLevel)
+
+	setInstanaLogLevel(l)
 
 	return l
 }
 
+func setInstanaLogLevel(l *Logger) {
+	instanaLogLevelEnvVar := os.Getenv("INSTANA_LOG_LEVEL")
+
+	envVarLogLevel := map[string]Level{
+		"error": ErrorLevel,
+		"info":  InfoLevel,
+		"warn":  WarnLevel,
+		"debug": DebugLevel,
+	}
+
+	if value, ok := envVarLogLevel[strings.ToLower(instanaLogLevelEnvVar)]; ok {
+		l.SetLevel(value)
+	} else {
+		l.SetLevel(ErrorLevel)
```

Comment:
@willianpc would it makes sense, to handle a case when value is provided but is invalid (write a message or something like this)? with the current implementation, we will silently set the level as an error, but it will be not visible to the user.

Reply:
I will create a separate PR for this improvement.